jwst/stpipe/tests/test_step.py

```diff
@@ -5,10 +5,128 @@
 )
 
 import pytest
-
+import jwst
 from jwst import datamodels
+from jwst.refpix import RefPixStep
 from jwst.stpipe import Step
 from jwst.stpipe.config_parser import ValidationError
+from jwst.stpipe import crds_client
+from jwst.extern.configobj.configobj import ConfigObj
+
+from .steps import MakeListStep
+from .util import t_path
+
+from crds.core.exceptions import CrdsLookupError
+
+
+ParsModelWithPar3 = datamodels.StepParsModel(t_path(join('steps','jwst_generic_pars-makeliststep_0002.asdf')))
+ParsModelWithPar3.parameters.instance.update({'par3': False})
+
+REFPIXSTEP_CRDS_MIRI_PARS = {
+    'class': jwst.refpix.refpix_step.RefPixStep,
+    'name': 'refpix',
+    'odd_even_columns': False,
+    'odd_even_rows': False,
+    'side_gain': 10.0,
+    'side_smoothing_length': 21,
+    'use_side_ref_pixels': False
+}
```

Comment:
This is only referenced in the test below.  Perhaps it should live in the test below and not be a global variable?

Reply:
Maybe just one test today, but may be used by many tomorrow. *shrug* either way.